e2e/test/scenarios/embedding/embedding-full-app.cy.spec.js

```diff
@@ -18,23 +23,26 @@ describe("scenarios > embedding > full app", () => {
 
   describe("home page navigation", () => {
     it("should hide the top nav when nothing is shown", () => {
-      visitUrl({ url: "/", qs: { side_nav: false, logo: false } });
+      visitFullAppEmbeddingUrl({
```

Comment:
some diffs are different from the original PR https://github.com/metabase/metabase/pull/33337, but what the original PR did is only renaming `visitUrl` to `visitFullAppEmbeddingUrl` which I also do it here.